File: 2114-maximum-number-of-words-found-in-sentences/2114-maximum-number-of-words-found-in-sentences.cpp
class Solution {
public:
    int mostWordsFound(vector<string>& sentences) {
        int maxx = 0;
        for(int i = 0; i < sentences.size(); i++){
            int count = std::count(sentences[i].begin(), sentences[i].end(), ' ');
            maxx = max(maxx, count + 1);
        }
        return maxx;
    }
};

File: 2114-maximum-number-of-words-found-in-sentences/2114-maximum-number-of-words-found-in-sentences.java
class Solution {
    public int mostWordsFound(String[] s) {
        int max = 0;
        for(int i = 0; i < s.length; i++){
            int c = 0;
            for(int j = 0; j < s[i].length(); j++){
                if(s[i].charAt(j) == ' ') c++;
            }
            max = Math.max(max,c+1);
        }
        return max;
    }
}